contribs/emissions/src/main/java/org/matsim/contrib/emissions/WarmEmissionHandler.java

```diff
@@ -144,7 +161,7 @@ public void handleEvent(LinkLeaveEvent event) {
 		double linkLength = link.getLength();
 		double leaveTime = event.getTime();
 
-		if ( !warnIfZeroLinkLength( linkId, linkLength ) ) { linkLeaveCnt++; }
+		if ( warnIfZeroLinkLength( linkId, linkLength ) ) { linkLeaveCnt++; }
 		// excluding links with zero lengths from leaveCnt. Amit July'17
```

Comment:
Do we need this if statement here anymore?
I think, that we NOW just can do the `linkLeaveCnt++`, or?